pyomo/contrib/pynumero/algorithms/solvers/cyipopt_solver.py

```diff
@@ -563,7 +292,7 @@ class PyomoCyIpoptSolver(object):
         "intermediate_callback",
         ConfigValue(
             default=None,
-            description="Set the function that will be called each iteration.",
+            description="Set the function that will be called each" " iteration.",
```

Comment:
Is this part of black's string normalization?

Reply:
This appears to be what happens when black combines a two-line string onto a single line. I think combining the strings would be "altering your code," so black doesn't do it.

Reply:
Yeah, we saw this happen in most of the PRs applying black and I think Miranda went through and manually removed the extra quotation marks.